Sources/LiveViewNative/ContentBuilder.swift

```diff
@@ -0,0 +1,533 @@
+    /// If no custom type eraser is available, unbox the content and erase the `View` that holds it.
+    ///
+    /// ```swift
+    /// func unbox(_ content: some MapContent) -> AnyView {
+    ///     AnyView(Map {
+    ///         content
+    ///     })
+    /// }
+    ///
+    /// let content: any MapContent = ...
+    /// unbox(content)
+    /// ```
+    ///
+    /// If a custom type eraser *is* available, it may still be preferable to use an `any` existential.
+    /// This can avoid creating many wrapper types when just one can be used at the root of the hierarchy.
+    ///
+    /// ```swift
+    /// let content: any ChartContent = ...
+    /// Chart {
+    ///     AnyChartContent(content)
+    /// }
+    /// ```
+    associatedtype Content
+    
+    /// Switch over the ``tag`` and provide the appropriate ``Content``.
+    ///
+    /// The ``ObservedElement`` and ``LiveContext`` property wrappers cannot be used outside of `View`.
+    /// Therefore, the ``element`` and ``context`` must be passed manually into each type to be used.
+    static func lookup<R: RootRegistry>(
+        _ tag: TagName,
+        element: ElementNode,
+        context: Context<R>
+    ) -> Content
+    
+    /// Provide an empty content, such as `EmptyView` in SwiftUI.
+    static func empty() -> Content
+    
+    /// Merge content together, such as `TupleView<A, B>` in SwiftUI.
+    static func reduce(
+        accumulated: Content,
+        next: Content
+    ) -> Content
+    
+    /// Switch over the ``type`` and provide the appropriate ``ContentModifier``.
+    ///
+    /// The `RootRegistry` type is included so modifiers can build nested content.
+    static func decodeModifier<R: RootRegistry>(
+        _ type: ModifierType,
+        from decoder: Decoder,
+        registry _: R.Type
+    ) throws -> any ContentModifier<Self>
+}
+
+public extension ContentBuilder {
+    /// The ``context`` associated with this builder.
+    ///
+    /// This is used to support the building of nested `View` elements from within any builder
+    /// with the ``buildView(_:in:)`` function.
+    typealias Context<R: RootRegistry> = ContentBuilderContext<R>.Value
+}
+
+/// Access the context for a ``ContentBuilder`` from within a `View`.
+///
+/// Use this property wrapper in a `View` to access the ``ContentBuilder.Context`` from the environment.
+///
+/// ```swift
+/// struct MyChart<R: RootRegistry>: View {
+///   @ContentBuilderContext<R> private var context
+///   @ObservedElement private var element
+///
+///   var body: some View {
+///     Chart {
+///       ChartContentBuilder.build(
+///         element.children(),
+///         in: context
+///       )
+///     }
+///   }
+/// }
+/// ```
+@propertyWrapper
+public struct ContentBuilderContext<R: RootRegistry>: DynamicProperty {
+    @Environment(\.coordinatorEnvironment) private var coordinatorEnvironment
+    @LiveContext<R> private var context
+    
+    public init() {}
```

Comment:
@carson-katri Did you mean to leave this as is?

Reply:
Yeah, I need the init to be public but don't need any arguments to it.